src/specs/components/mega-menu/demo/data.js

```diff
@@ -32,15 +32,13 @@ module.exports = {
     {
       label: 'News and media',
       path: exampleLink,
-      trigger_aria_label: 'Access news and media children',
```

Comment:
this parameter was added in order to be able to provide the screen reader users the information about the fact that the "trigger" is only a button in the UI, clicking on it is not moving the user anywhere while the first item accessed after the "click" is again a link with the same text and this time this is really a link.
Might not have been the best way to make this distinction clear, but it seems to me that removing it will create a pretty confusing experience for the user. (this thing will change, by the way, not sure it would be relevant also in the updated version of the menu)

Reply:
ideally the first level items should be buttons instead of links, as they only open the dropdown. This would remove the confusion.
anyways, let's see that in the "updated" version